README.rst

```diff
@@ -503,8 +503,83 @@ Models
+Often these are known as second-level or group-level analyses.
+There are currently two ways in which to run these analyses, either through
+the user interface, or using a ‘group-level’ command directly in the terminal.
+Shown first is the user interface method.
+
+#. ``Specify group-level model?``
+
+   -  ``Yes``
+
+      #. ``Specify model type``
+
+         -  ``Intercept-only`` This will apply an intercept-only model that captures a main effect of one or more of the first-level features.
+
+            #. ``Specify model name``
+            #. ``Aggregate scan-level statistics before analysis?`` This option may appear if there are multiple subject-level statistical images created, such as for multiple runs or sessions. If this option is selected (with an asterisk) then these features will be aggregated at the subject-level prior to group-level analysis. If the option is not selected, then all of the statistical images of each subject’s first-level features will be entered into the group-level analysis, treating them as if they were independent observations from different subjects, therefore this option is generally not preferred.
+            #. ``Exclude subjects based on movement?``
+
+               -  ``Yes``
+
+                  #. ``Specify the maximum allowed mean framewise displacement in mm`` By selecting 0.5, this will exclude all subjects (or runs/sessions for a subject if ‘aggregation of scan-level statistics before analysis’ is not selected above) with a mean framewise displacement (root-mean-square) >0.5 mm. If aggregation of scan-level statistics before analysis is selected, the aggregate image will be used in the group-level analyses, therefore the average framewise displacement of all images used for the aggregate image will be compared to the value set here – if it is exceeded, the subject will be excluded from analysis.
+                  #. ``Specify the maximum allowed percentage of frames above the framewise
+                     displacement threshold of (FD max from above)``
+
+               -  ``No`` All subjects are retained for analyses regardless of movement.
+
+         -  ``Linear model`` This will apply a linear model that captures main and/or interaction effects of one or more variables that must be present in a covariate/group data spreadsheet file. This file must contain a column with subject ID names identical to those used for the functional data.
+
+            #. ``Specify model name``
+            #. ``Aggregate scan-level statistics before analysis?`` See above
+            #. ``Specify the path of the covariates/group data spreadsheet file`` Can be a .csv, .txt, .tsv, or .xlsx file
+
+               #. ``Specify the column containing subject names``
+               #. ``Specify the column data types`` Note that the black option is selected, the grey option is not. Toggle left/right keys to select or deselect options and up/down keys to move through the list of variables in your spreadsheet file.
```

Comment:
It would be great to have some additional details on how the different data types are handled. In short, categorical are dummy coded and continuous are demeaned

Reply:
Resolved in section below 'Specify the variables to add to the model'